### PR TITLE
Add AWS CLI to docker container to get secrets on startup

### PR DESCRIPTION
**Pull request by ajmaddaford**
### What is the context of this PR?
Adds the AWS CLI to the docker container so it can get the secrets from S3 on startup.

### How to review 
- Build and run the docker container
- Verify the `aws` command is working in the container e.g. `docker exec -ti eqsurveyrunner_eq-survey-runner_1 aws`

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,7 @@
 FROM python:3.4
 
-RUN pip install -U pipenv==8.2.7
+RUN pip install -U pipenv==8.2.7 \
+  && pip install -U awscli==1.11.174
```
Comment (samiwel) [inline Dockerfile:4]: Why are the versions specified in-line here rather than in a requirements.txt file?

Comment (jonnyshaw89) [reply Dockerfile:4]: The aren't dependencies of the application. they are dependencies of the docker environment

**Review by jonnyshaw89 — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (ajmaddaford) [reply Dockerfile:4]: Because these are dependencies required to bootstrap survey runner - they aren't required to run survey runner.

Comment (samiwel) [reply Dockerfile:4]: gotchya

**Review by MIslam92 — APPROVED** [review]

**Pull request merged**